cvcl/models.py

```diff
@@ -1,3 +1,27 @@
 from django.db import models
+from django.utils import timezone
 
-# Create your models here.
+class User(models.Model)
+    username = models.CharField(max_length=30)
+    password = models.CharField(max_length=30)
```

Comment:
We don't need passwords in our database since the authentication is handled externally. Think?